src/mmw/js/src/main.js

```diff
@@ -22,6 +22,12 @@ L.Icon.Default.imagePath = '/static/images/';
 var csrf = require('./csrf');
 $.ajaxSetup(csrf.jqueryAjaxSetupOptions);
 
+var ZeroClipboard = require('zeroclipboard');
+ZeroClipboard.config({
+    hoverClass: 'focus',
+    activeClass: 'active'
+});
+
```

Comment:
Will rebase on top of #319 after it has been merged and move this to new `setup.js`